Composer/packages/client/src/components/TestController/emulatorOpenButton.tsx

```diff
@@ -4,30 +4,43 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/core';
 import formatMessage from 'format-message';
-import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { ActionButton, IconButton } from 'office-ui-fabric-react/lib/Button';
+import { TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
+import { Fragment } from 'react';
 
 import { BotStatus } from './../../constants';
 
 interface IEmulatorOpenButtonProps {
+  botEndpoint: string;
   botStatus: BotStatus;
   hidden: boolean;
   onClick: () => void;
 }
 
 export const EmulatorOpenButton: React.FC<IEmulatorOpenButtonProps> = props => {
-  const { onClick, botStatus, hidden } = props;
+  const { onClick, botStatus, hidden, botEndpoint } = props;
   const connected = botStatus === BotStatus.connected;
 
   if (hidden || !connected) return null;
 
   return (
-    <ActionButton
-      iconProps={{
-        iconName: 'OpenInNewTab',
-      }}
-      onClick={onClick}
+    <TooltipHost
+      content={
+        <Fragment>
+          URL: {botEndpoint}{' '}
```

Comment:
Not a fan of this URL: prefix. Can we just remove it and show botEndpoint?

Reply:
Thought it would be more like the Azure style, there is a key:value, but not sure, removing them is also good, done